EVENT.md

```diff
@@ -783,10 +783,10 @@ included in the request body as a JSON list of objects, e.g.:
 
 ```jsonc
 [{
-  "type": "<report type>", // e.g. "source-destination-limit"
+  "type": "<report type>", // e.g. "source-destination-rate-limit"
   "body": {
     "attribution_destination": "https://destination.example",
-    "limit": "100", // the browser's limit
+    "limit": "50", // the browser's limit
```

Comment:
Why are we changing this? It's just an example.

Reply:
As we need to change the report types (`source-destination-limit` deprecated`), I think it's clearer to update the limit as well.

Reply:
But this is just an example. The value is meaningless.